.github/workflows/PRWorkflow.yml

```diff
@@ -0,0 +1,23 @@
+name: Slack pull request open notification
+
+on:
+  pull_request:
+    types: [review_requested, ready_for_review]
+
+jobs:
+  Notify:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Notify slack pr ready for review
+      env: 
+        SLACK_WEBHOOK_URL : ${{ secrets.SLACK_WEBHOOK_URL }}
+        PULL_REQUEST_NUMBER : ${{ github.event.pull_request.number }}
+        PULL_REQUEST_TITLE : ${{ github.event.pull_request.title }}
+        PULL_REQUEST_AUTHOR_NAME : ${{ github.event.pull_request.user.login }}
+        PULL_REQUEST_AUTHOR_ICON_URL : ${{ github.event.pull_request.user.avatar_url }}
+        PULL_REQUEST_URL : ${{ github.event.pull_request.html_url }}
+        PULL_REQUEST_BODY : ${{ github.event.pull_request.body }}
+        PULL_REQUEST_COMPARE_BRANCH_NAME : ${{ github.event.pull_request.head.ref }}
+        PULL_REQUEST_BASE_BRANCH_NAME : ${{ github.event.pull_request.base.ref }}
+        IS_SEND_HERE_MENTION : true
```

Comment:
pls do not @here for every PR going out for review thx :)

Reply:
okay fixed